draft-ietf-6man-pio-pflag.xml

```diff
@@ -186,26 +183,32 @@ already running.</li>
 stop DHCPv6 prefix delegation if it has no other reason to run it.
 The lifetimes of any prefixes already obtained via DHCPv6 are unaffected.</li>
 <li>
-If the client has already received delegated prefix(es) from one or more
-servers, then any time a prefix is added to or removed from the list, the client MUST
+If the host has already received delegated prefix(es) from one or more
+servers, then any time a prefix is added to or removed from the list, the host MUST
 consider this to be a change in configuration information as
 described in <xref target="RFC8415" section="18.2.12"/>, and it MUST perform
-a REBIND, unless it is going to stop the client because the list became empty.
+a REBIND, unless it is going to stop the DHCPv6 client because the list became empty.
 This is in addition to performing a REBIND in the other cases required by that
 section. Issuing a REBIND allows the host to obtain new prefixes if necessary, e.g.,
 when the network is being renumbered. It also refreshes state related to the
 delegated prefix(es).
 </li>
 </ul>
 <t>
-When a host requests a prefix via DHCPv6 PD, it MUST use the prefix length hint  <xref target="RFC8415" section="18.2.4"/> to request a prefix that is short enough to form addresses via SLAAC.
+When a host requests a prefix via DHCPv6 PD, it MUST use the prefix length hint <xref target="RFC8415" section="18.2.4"/> to request a prefix that is short enough to form addresses via SLAAC.</t>
 
-	 </t>
-	   
+<t>
+In order to achieve the scalability benefits of using DHCPv6 PD, the host SHOULD prefer to form addresses from the delegated prefix instead of using individual addresses in the on-link prefixes. Therefore, when the host requests a prefix using DHCPv6 PD:
+</t>
+<ul>
+<li>It SHOULD NOT use SLAAC to obtain IPv6 addresses from prefix(es) with the P bit set.</li>
+<li>It SHOULD NOT request individual IPv6 addresses from DHPCv6, i.e., it SHOULD NOT include any IA_NA or IA_TA options in Solicit, Renew or Rebind messages.</li>
+</ul>
 	   
-	   <t>
-The P flag is meaningless for link-local prefixes and any Prefix Information Option containing the link-local prefix MUST be ignored as specified in <xref target="RFC4862" section="5.5.3"/>.
+<t>If the host doesn't obtain any suitable prefixes via DHCPv6 PD it MAY choose, as a fallback mechanism, to form addresses via SLAAC using PIOs with A flag set to 1.</t>
```

Comment:
Shall we say 'fallback to other address assignment mechanisms available to the host, such as SLAAC' to address potential concern of the IA_NA funclub? ;)

Reply:
Reworded. Better?